### PR TITLE
[#113] Elements no longer highlightable

### PR DESCRIPTION
**Pull request by kammanz**
Fixed the elements issue and did the remaining V1 issues in this branch.

What's completed: 
- Elements no longer highlightable
- [#114] Removed pointer and pointer events on selected player card (and also nav link)
- [#116] Added under construction pages for games and standings, created gif folder in assets, with lebron gif
- [#80] Fixed scroll bar bug on player details card

### CODE REVIEW

**Comment by kammanz** [comment]
Ready for review.

**Comment by kammanz** [comment]
Ready for review.

File: src/components/games/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import 'src/scss/variables.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: calc(100vh - 90px);
```
Comment (Tumne) [inline src/components/games/index.module.scss]: `height: calc(100vh - 45px);`

File: src/components/standings/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import 'src/scss/variables.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 1;
+  justify-content: center;
+  align-items: center;
+  height: calc(100vh - 90px);
```
Comment (Tumne) [inline src/components/standings/index.module.scss]: `height: calc(100vh - 45px);`

File: src/components/standings/index.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 
+import lebron from 'assets/gifs/lebron.gif';
```
Comment (Tumne) [inline src/components/standings/index.js]: rename
```
import standingsGif from 'assets/gifs/standings.gif';
```

File: src/components/games/index.js [hunk 4]
```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import lebron from 'assets/gifs/lebron.gif';
```
Comment (Tumne) [inline src/components/games/index.js]: rename
```
import gamesGif from 'assets/gifs/games.gif';
```

where this should be the gif: 

https://media3.giphy.com/media/5n1XqQh0DskgKMKejY/giphy.gif

**Review by Tumne — CHANGES_REQUESTED** [review]
Looks good, Minor fixes and some design changes, (adding another gif image).

**Comment by kammanz** [comment]
Ready for review.

File: src/components/header/navMenu.js [hunk 5]
```diff
@@ -11,6 +11,10 @@ const NavMenu = ({
   selectedTeam: { teamColor, urlName },
   playerId,
 }) => {
+  const activeStyle = {
+    pointerEvents: 'none',
+    color: teamColor,
+  };
```
Comment (Tumne) [inline src/components/header/navMenu.js:17]: I believe this was meant for PR #130, whoops again.
Try to keep your PRs clean when switching between them.

**Review by Tumne — COMMENTED** [review]
A reminder

File: src/components/header/navMenu.js (hunk 5 above)
Comment (Tumne) [reply src/components/header/navMenu.js:17]: nvm, I see what you did, consider resolved

**Review by Tumne — APPROVED** [review]
Approved and completed minor UI changes

**Pull request merged**